app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -5,6 +5,9 @@ class LtiConfigurationsController < Orgs::Controller
     before_action :ensure_lti_launch_flipper_is_enabled
     before_action :ensure_current_lti_configuration, except: %i[new create]
 
+    skip_before_action :authenticate_user!, only: :autoconfigure
+    skip_before_action :ensure_current_organization_visible_to_current_user, only: :autoconfigure
```

Comment:
I'm wondering if `#autoconfigure` makes sense within the `#lti_configurations_controller` mainly because whenever the LMS makes its initial request to Classroom to generate a configuration file it will be a plain old non-authenticated GET request, meaning the general assumptions around a subclass of `Orgs::Controller` don't fully apply. 

My followup question would be: If not in this controller, where?